src/student/student.service.ts

```diff
@@ -3,21 +3,96 @@ import { CreateStudentDto } from './dto/create-student.dto';
 import { UpdateStudentDto } from './dto/update-student.dto';
 import { ReadStudentDto } from './dto/read-student.dto';
 import { PrismaService } from 'src/prisma/prisma.service';
+import { createClient } from '@supabase/supabase-js';
+import { decode } from 'base64-arraybuffer';
+import { v4 as uuidv4 } from 'uuid';
 
 @Injectable()
 export class StudentService {
   constructor(private prisma: PrismaService) {}
 
-  create(createStudentDto: CreateStudentDto) {
-    const testStudent = this.prisma.testModel.create({
+  getPaymentById(createStudentDto) {
```

Comment:
not sure if this function is used somewhere, if wala pa remove nalang din

Reply:
Resolved, thanks!
Action:
-Remove getPaymentById()